wcivf/settings/base.py

```diff
@@ -42,8 +43,32 @@ def repo_root(*x):
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = False
 
-# add to param store?
-ALLOWED_HOSTS = ["*"]
+# Hosts/domain names that are valid for this site; required if DEBUG is False
+# See https://docs.djangoproject.com/en/1.5/ref/settings/#allowed-hosts
+ALLOWED_HOSTS = [
+    os.environ.get("FQDN", None),
+    "localhost",
+    "127.0.0.1",
+]
+
+
+def get_ec2_ip():
```

Comment:
I guess we could consider making this a shared function in a package? e.g: dc_django_utils. I think for now doing it right with a bit of copy & paste is a step forward.